src/cljc/matcher_combinators/core.cljc

```diff
@@ -397,9 +398,11 @@
       {::result/type   :match
        ::result/value  elements
        ::result/weight 0}
-      (match (->EqualsSeq (concat (:matched result)
-                                  (:unmatched result)))
-             (:elements result)))))
+      (update (match (->EqualsSeq (concat (:matched result)
+                                          (:unmatched result)))
+                     (:elements result))
+              ::result/value
+              #(with-mismatch-meta % :mismatch-sequence)))))
```

Comment:
add `:mismatch-sequence` metadata, which is used by the elision feature, to the result of any-order/set mismatch results